scifight/models.py

```diff
@@ -27,9 +41,11 @@ class Meta:
 
 
 class Team(models.Model):
-    name = models.CharField(max_length=NAME_LENGTH)
+    tournament  = models.ForeignKey(Tournament)
+    name        = models.CharField(max_length=NAME_LENGTH)
     description = models.TextField(max_length=TEXT_LENGTH, blank=True)
-    origin = models.ForeignKey(TeamOrigin, null=True, blank=True)
+    origin      = models.ForeignKey(TeamOrigin, null=True, blank=True)
+    slug        = models.SlugField(max_length=SLUG_LENGTH, null=True, blank=True, unique=True)
```

Comment:
I'd like to order those fields like: `tournament`, `name`, `slug`, `description`, `origin`.